improver/calibration/__init__.py

```diff
@@ -101,8 +102,8 @@ def split_forecasts_and_truth(
         land_sea_mask = None
     elif len(land_sea_mask) != 1:
         msg = (
-            "Expected at most one cube for land-sea mask. The number of land-sea "
-            "masks provided was {len(land_sea_mask)}."
+            f"Expected at most one cube for land-sea mask. The number of land-sea "
```

Comment:
Just to note, this first `f` isn't required, as the `f` is only required at the start of the string containing the variable to be filled. The additional `f` could also be removed on line 119.

Reply:
Done